test/b-splines/linear.jl

```diff
@@ -28,14 +28,14 @@ for (constructor, copier) in ((interpolate, x->x), (interpolate!, copy))
 
     # 2D
     g2(y) = cos(y/6)
-    f(x,y) = g1(x)*g2(y)
```

Comment:
This should probably be resolved instead by moving the definitions of `f(x,y)` out of the loop. To do so, move `g2`, `ymax` and `A2``too - but all of these should probably have been outside of the loop to begin with...